serverpool/src/main/java/com/ats/serverpool/Message.java

```diff
@@ -5,6 +5,11 @@
     private String messageType;
     private String message;
 
+    public Message() {
```

Comment:
@aleksajovanovic I added a constructor instanciating an empty Message object. Just in case I added the setter methods as well.